presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSessionProperties.java

```diff
@@ -138,6 +139,15 @@ public PrestoSparkSessionProperties(PrestoSparkConfig prestoSparkConfig)
                         "When revoking memory, try to revoke so much that memory pool is filled below target at the end",
                         prestoSparkConfig.getMemoryRevokingTarget(),
                         false),
+                new PropertyMetadata<>(
+                        SPARK_EXECUTION_STRATEGIES,
+                        "Execution strategies to be applied while running the query",
+                        VARCHAR,
+                        List.class,
+                        prestoSparkConfig.getExecutionStrategies(),
```

Comment:
I don't see a use case to have a config property for this session property. Do you see any valid use case for this? If not, we can just  set the default value to Empty list.

Reply:
I dont see any valid use case. Reverted config property.